expressEngine/lib/responseHandler/responseHandler.js

```diff
@@ -0,0 +1,17 @@
+import { error as commonError, success } from '../../cms/common';
+
+class ResponseHandler {
+  success = ({ data = true, message = success.defaultSuccessMsg }) => ({ data, message });
+
+  error = ({ error = commonError.undefinedError, status = 500 }) => ({ error, status });
+
+  response = (res, data) => {
+    const { error } = data;
+    if (error) {
+      res.json(this.error(data));
```

Comment:
@chiragSuccessive Set status in response object. You don't need to send it separately.